src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Contracts/Payments/ListPaymentsResponse.cs

```diff
@@ -0,0 +1,6 @@
+namespace TicketManager.PaymentService.Contracts.Payments;
+
+public class ListPaymentsResponse
+{
+    public List<PaymentDto> Payments { get; set; }
+}
```

Comment:
You don't need this Response. You can return `List<PaymentDto>` from endpoint.

Reply:
Addressed in `8a0f91ef`.